modules/appeals_api/app/serializers/appeals_api/evidence_submission_serializer.rb

```diff
@@ -1,13 +1,30 @@
 # frozen_string_literal: true
 
+require_dependency 'common/exceptions'
 require 'fast_jsonapi'
 
 module AppealsApi
-  class EvidenceSubmissionSerializer
-    include FastJsonapi::ObjectSerializer
+  class EvidenceSubmissionSerializer < ActiveModel::Serializer
+    type 'evidence_submission'
 
-    set_key_transform :camel_lower
-    attributes :status
-    set_type :evidenceSubmission
+    attributes :status, :code, :details, :appeal_type, :appeal_id, :location
+
+    delegate :code, :details, to: :object
```

Comment:
Delegating `details` means it could return a large amount of text from an error backtrace or something. Let's make sure it only returns the first 100 characters of the details by having the serializer truncate the output.

Also it should be `detail`, not `details`, to follow our other models - but that's a fix for another ticket :P

Reply:
Yeah, the migration set `details`, so a ticket is in order